pkg/admission/webhook/generic_webhook.go

```diff
@@ -87,9 +86,10 @@ type WebhookDispatcher struct {
 	dispatcher generic.Dispatcher
 	hookSource ClusterAwareSource
 
-	getAPIExport func(path logicalcluster.Path, name string) (*apisv1alpha1.APIExport, error)
+	getAPIExport func(path logicalcluster.Name, name string) (*apisv1alpha1.APIExport, error)
 
 	apiBindingClusterLister apisv1alpha1listers.APIBindingClusterLister
+	apiExportClusterLister  apisv1alpha1listers.APIExportClusterLister
```

Comment:
We're moving away from storing listers/informers/indexers as struct fields. The only thing we need on the struct is the get/list* function pointer.